ee/query-service/anomaly/daily.go

```diff
@@ -24,9 +27,18 @@ func NewDailyProvider(opts ...GenericProviderOption[*DailyProvider]) *DailyProvi
 		opt(dp)
 	}
 
+	dp.querierV2 = querierV2.NewQuerier(querierV2.QuerierOptions{
+		Reader:        dp.reader,
+		Cache:         dp.cache,
+		KeyGenerator:  queryBuilder.NewKeyGenerator(),
+		FluxInterval:  dp.fluxInterval,
+		FeatureLookup: dp.ff,
```

Comment:
log new schema flag

Reply:
We do not support anomalies on logs or traces. Should I still add flag?

Reply:
not required